src/openforms/forms/tests/test_api_formdefinition.py

```diff
@@ -156,6 +156,35 @@ def test_update(self):
         self.assertEqual(True, definition.login_required)
         self.assertIn({"label": "New field"}, definition.configuration["components"])
 
+    def test_update_is_reusable_unsuccessful_with_multiple_forms(self):
+        user = StaffUserFactory.create(user_permissions=["change_form"])
+        self.client.force_authenticate(user=user)
+
+        step = FormStepFactory.create(
+            form__internal_name="InternalForm",
+            form_definition__internal_name="InternalDef",
+            order=1,
+        )
```

Comment:
```suggestion
        step = FormStepFactory.create()
```
* form name is not relevant for the test
* form definition name is not relevant for the test
* order is not relevant
* order starts at `0`
* order is set implicitly by `django-ordered-model` anyway